src/components/shell/header/header.container.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { connect } from 'react-redux';
+import { translate } from 'react-i18next';
```

Comment:
Do you need translate?  If Header was relying on the t() function before, where was it getting it from? Do those locations need to be cleaned up?